src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/UpdateDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,573 @@
+Feature: Resources "Update" Operation validations
+
+    Rule: Descriptors
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And the system has these descriptors
+                  | descriptorValue                                                                            |
+                  | uri://ed-fi.org/ContentClassDescriptor#Testing                                             |
+                  | uri://ed-fi.org/AddressTypeDescriptor#Physical                                             |
+                  | uri://ed-fi.org/StateAbbreviationDescriptor#TX                                             |
+                  | uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider |
+                  | uri://ed-fi.org/GradeLevelDescriptor#Postsecondary                                         |
+                  | uri://ed-fi.org/AbsenceEventCategoryDescriptor#Sick Leave                                  |
```

Comment:
You probably don't need any of these, just the POST request below here.

Reply:
Removed